itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -838,7 +859,26 @@ public void testIncrementalAdds() throws IOException {
     verifySetup("SELECT a from " + dbName + ".ptned_late WHERE b=2", ptn_data_2, driver);
 
     // Perform REPL-DUMP/LOAD
-    incrementalLoadAndVerify(dbName, replDbName);
+    Tuple incrementalDump = incrementalLoadAndVerify(dbName, replDbName);
+    FileSystem fs = new Path(bootstrapDump.dumpLocation).getFileSystem(hconf);
+    boolean dumpAckFound = false;
+    boolean loadAckFound = false;
+    assertFalse(fs.exists(new Path(bootstrapDump.dumpLocation)));
+    fs = new Path(incrementalDump.dumpLocation).getFileSystem(hconf);
+    Path dumpPath = new Path(incrementalDump.dumpLocation, ReplUtils.REPL_HIVE_BASE_DIR);
+    dumpAckFound = false;
+    loadAckFound = false;
+    for (FileStatus status : fs.listStatus(dumpPath)) {
+      if (status.getPath().getName().equalsIgnoreCase(ReplUtils.DUMP_ACKNOWLEDGEMENT)) {
+        dumpAckFound = true;
+      }
+      if (status.getPath().getName().equalsIgnoreCase(ReplUtils.LOAD_ACKNOWLEDGEMENT)) {
+        loadAckFound = true;
+      }
+    }
+
+    assertTrue(dumpAckFound);
+    assertTrue(loadAckFound);
```

Comment:
Why not just use:
assertTrue(fs.exists(new Path(dumpPath, ReplUtils.DUMP_ACKNOWLEDGEMENT)));
assertTrue(fs.exists(new Path(dumpPath, ReplUtils.LOAD_ACKNOWLEDGEMENT)));